client/app/visualizations/sankey/initSankey.js

```diff
@@ -0,0 +1,237 @@
+import { isNil, map, extend, sortBy, includes, filter, reduce, find, keys, values, identity } from 'lodash';
+import d3 from 'd3';
+import d3sankey from './d3sankey';
+
+function getConnectedNodes(node) {
+  // source link = this node is the source, I need the targets
+  const nodes = [];
+  node.sourceLinks.forEach((link) => {
+    nodes.push(link.target);
+  });
+  node.targetLinks.forEach((link) => {
+    nodes.push(link.source);
+  });
+
+  return nodes;
+}
+
+function graph(data) {
+  const nodesDict = {};
+  const links = {};
+  const nodes = [];
+
+  // ANGULAR_REMOVE_ME $$ check is for Angular's internal properties
```

Comment:
Not redundant now?

Reply:
No - sankey may still receive data from Angular part, so these properties may be present there. It will be safe to remove when we'll completely remove Angular itself.